static/espeasy_default.css

```diff
@@ -34,7 +27,7 @@ h6 {
 * {
 	box-sizing: border-box;
 	font-family: sans-serif;
-	font-size: 12pt;
+	/*font-size: 12pt;*/
```

Comment:
Just curious... When minifying the css, are commented parts removed?

Reply:
yep, of course, they are..

Reply:
BTW your wonderfull embed_file.sh script does NOT generate the minified version of the CSS into the "espeasy_default.min.css" file .....

Reply:
> BTW your wonderfull embed_file.sh script does NOT generate the minified version of the CSS into the "espeasy_default.min.css" file .....

It used to do, but did it with uploading to some online service.
So what I did the last few times is to minify it myself using copy/paste to some online minify service.

Reply:
I've made some fixes in #3885, to regerate this file, but also to correctly minimize the CSS that were using an HTML-minifier which did not minify CSS enought. 

I will thus update this current CSS build to now use the righly compressed PROGMEN blob